collect/static/css/styles.css

```diff
@@ -1,9 +1,10 @@
+@import url("https://fonts.googleapis.com/css?family=Fira Sans");
```

Comment:
@Abhishek-Deshmukh Is this font available to download? The webapp will run locally, perhaps without internet connection.